src/components/inputs/toggle-input/toggle-input.js

```diff
@@ -88,42 +89,62 @@ const Input = styled.input`
   }
 `;
 
+const getToggleInputProps = (props = {}) => ({
+  type: 'checkbox',
+  checked: props.checked || props.isChecked,
+  disabled: props.disabled || props.isDisabled,
```

Comment:
At some point we'll have `toIsOrNotBooleanCase`-fn which expands a `disabled` to `isDisabled` and `disabled` (or any other prop) to spread it :)